peernode/src/peer-node/store/mod.rs

```diff
@@ -1,19 +1,31 @@
-use crate::peer::MarketClient;
 use crate::producer;
-use crate::transfer::files;
-use crate::transfer::jobs::Jobs;
-use anyhow::Result;
+use crate::transfer::files::LocalFileInfo;
+use crate::transfer::{
+    files::{self, get_file_info},
+    jobs::{self, Jobs},
+};
+use crate::{consumer::encode::EncodedUser, peer::MarketClient};
+use anyhow::{anyhow, Result};
+use async_recursion::async_recursion;
 use config::{Config, File, FileFormat};
 use orcanet_market::{BootNodes, Multiaddr};
+use proto::market::{FileInfoHash, User};
 use serde::{Deserialize, Serialize};
-use std::{collections::HashMap, fs, path::PathBuf};
+use std::{
+    collections::HashMap,
+    fs,
+    path::{Path, PathBuf},
+};
 
 #[derive()]
```

Comment:
Actually just one more thing, but think we can just remove this empty `#[derive()]` (it's not from your changes, but wondering if you can just remove that)